src/main/java/nl/tudelft/contextproject/model/level/RandomLevelFactory.java

```diff
@@ -7,17 +7,17 @@
 import java.util.regex.Pattern;
 
 import com.jme3.light.Light;
-import com.jme3.light.PointLight;
-import com.jme3.math.ColorRGBA;
 import com.jme3.math.Vector2f;
-import com.jme3.math.Vector3f;
+import nl.tudelft.contextproject.util.Size;
 
 import nl.tudelft.contextproject.util.FileUtil;
-import nl.tudelft.contextproject.util.Size;
 
 /**
  * An implementation of a LevelFactory that creates a level randomly.
```

Comment:
please add a newline after this line, so the @ lines are seperated from the method description